docs/default.nix

```diff
@@ -39,12 +34,15 @@ let
     documentsDirectory = ./.;
     documentType = "book";
     chunkToc = ''
-    <toc>
-      <d:tocentry xmlns:d="http://docbook.org/ns/docbook" linkend="book-home-manager-manual"><?dbhtml filename="index.html"?>
-        <d:tocentry linkend="ch-options"><?dbhtml filename="options.html"?></d:tocentry>
-        <d:tocentry linkend="ch-release-notes"><?dbhtml filename="release-notes.html"?></d:tocentry>
-      </d:tocentry>
-      </toc>
+      <toc>
+        <d:tocentry xmlns:d="http://docbook.org/ns/docbook" linkend="book-nixvim-manual"><?dbhtml filename="index.html"?>
+          <d:tocentry linkend="ch-options"><?dbhtml filename="options.html"?></d:tocentry>
+          <d:tocentry linkend="ch-release-notes"><?dbhtml filename="release-notes.html"?></d:tocentry>
+        </d:tocentry>
+        </toc>
     '';
   };
-in docs.html
+in
+# TODO: Parse this json or something, since docbook isn't working (and it's kind of terrible anyway)
```

Comment:
Have you considered alternative options for generating docs? Here are a few that I've stumbled across:

- [coricamu](https://github.com/danth/coricamu/)
- [styx](https://github.com/divnix/styx/)

I have also seen some projects using `mdbook` ([example](https://github.com/divnix/std)).

Reply:
From what I can tell, those are generic static site generators and not necessarily made for nix modules, like `nmd` is.